tests/Benchmarks/Helpers/Server.cs

```diff
@@ -67,7 +66,6 @@ public static void Run(int numberOfRequests, int concurrentConnections, byte[] r
             Task.WaitAll(tasks);
 
             listener.Dispose();
-            memoryPool.Dispose();
```

Comment:
We no longer have to call dispose on memorypool? Is it because of the change to use the Shared MemoryPool rather than instantiating a new one? cc @AtsushiKan

Reply:
Yes, it's because we don't create it anymore.